packages/checkout/widgets-lib/src/widgets/bridge/components/BridgeForm.tsx

```diff
@@ -15,7 +15,7 @@ import {
 import {
   useCallback, useContext, useEffect, useMemo, useRef, useState,
 } from 'react';
-import { BigNumber, utils } from 'ethers';
+import { BigNumber, ethers, utils } from 'ethers';
```

Comment:
remove ethers from the import here. You can just use BigNumber.from

Reply:
Yea, thanks, updated.